include/benchmark/benchmark.h

```diff
@@ -583,7 +586,7 @@ typedef void(Function)(State&);
 class Benchmark {
  public:
   virtual ~Benchmark();
-
+  typedef std::map<std::string, std::string> MetaDataType;
```

Comment:
Not commenting on the typedef itself, but the placement seems not ideal, maybe move down to getters/setters?